pom.xml

```diff
@@ -435,7 +439,6 @@
                         <redirectTestOutputToFile>true</redirectTestOutputToFile>
                         <argLine combine.children="append">-Xmx1G
                             -Djava.net.preferIPv4Stack=true
-                            ${maven.surefire.arguments}
```

Comment:
There may be a tweak possible to make this line do the right thing, but I went down a different path.